WordPress/src/main/res/layout/reader_blog_info_view.xml

```diff
@@ -20,10 +20,18 @@
         android:paddingTop="@dimen/margin_large"
         tools:targetApi="LOLLIPOP">
 
+        <org.wordpress.android.widgets.WPNetworkImageView
+            android:id="@+id/image_blavatar"
+            style="@style/ReaderImageView.Avatar"
+            android:layout_centerVertical="true"
+            android:layout_marginRight="@dimen/margin_large"
+            tools:src="@drawable/blavatar_placeholder" />
+
```

Comment:
Not exactly an issue of this PR but, now that there is extra space used by the blavatar, would you consider anchoring the `text_blog_name` and `text_domain` to the left of the `follow_button`? As it is, a big name (and/or a big system font setting) can make these texts collide with the follow button. There _is_ such anchoring in the `layout/reader_cardview_post.xml` case.